<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vilmis-PC" failures="1" tests="1" name="Default test" time="2.183" errors="0" timestamp="06 birž. 2019 14:24:02 GMT">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.793">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepDefinitions.StepDefinition.response_contains_temperature_value_of_degrees_celsius(StepDefinition.java:91)
at ✽.Response contains temperature value of "28.15" degrees celsius(file:src/test/resources/features/bound_to_fail_feature/This_test_sould_fail.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
